packages/dataviews/src/components/dataform-context/index.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { FormField, NormalizedField } from '../../types';
+
+type DataFormContextType< Item > = {
+	getFieldDefinition: (
```

Comment:
In the DataViewsContext, we store the `fields` directly in the context, while here we provide an abstraction function. I wonder if we should be consistent and just keep `fields` here.

Reply:
Yeah I am happy to use `fields` here instead for consistency.